images/peddy/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This image adds cpg-hail, SM and peddy to the base image.
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.0
+
+ENV HAIL_QUERY_BACKEND service
+
+# hadolint ignore=DL3008, DL3013
+RUN apt-get update && apt-get install -y --no-install-recommends g++ libopenblas-base liblapack3 openjdk-11-jre-headless python3-pip && \
```

Comment:
I think python-3 and peddy versions should be pinned, the rest I'm happy to ignore

Reply:
Will do!